### ISSUE TITLE
Make it clearer that "Save as Note" needs to be the last step

### ISSUE THREAD

**Issue by chrisgrieser** [kevboh/longform#52]
On the Discord server, I noticed a few times that there was confusion about the use of the compile step "Save as Note" – many users did not use it as the last step and then were wondering why the last couple of steps weren't executed properly. 

It should be made clear in the description, that this step needs to be at the end.

**Comment by xjtusjtu** [comment]
Indeed, if I put "Remove Comments" behind "Save as Note", it doesn't work. Hope for improving it~~
